Another-application/Build-Container-Images.ps1

```diff
@@ -1,4 +1,4 @@
-# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
If we do need this app, please rename the folder to something more specific like IIS-application